cmd/root.go

```diff
@@ -44,6 +46,7 @@ in with avalanche subnet create myNewSubnet.`,
 	// Disable printing the completion command
 	rootCmd.CompletionOptions.HiddenDefaultCmd = true
 
+	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.avalanche-cli.json)")
```

Comment:
why not inside of .avalanche-cli ?

Reply:
We don't want users modifying that directory